python_modules/dagster/dagster/_core/host_representation/external.py

```diff
@@ -178,14 +180,74 @@ def _utilized_env_vars(self) -> Mapping[str, Sequence[EnvVarConsumer]]:
     def get_utilized_env_vars(self) -> Mapping[str, Sequence[EnvVarConsumer]]:
         return self._utilized_env_vars
 
+
+    def get_default_automation_policy_sensor_name(self):
+        repository_origin = self.get_external_origin()
+        return (
+            repository_origin.get_label()
+            if repository_origin.repository_name != SINGLETON_REPOSITORY_NAME
+            else repository_origin.code_location_origin.location_name
+        )
```

Comment:
I would lean towards "default_automation_policy_sensor"

Reply:
I recall @braunjj having some opinions that it would be better to vary them, after some scarring experiences with a bunch of asset groups all called "default". What's the best way to move that conversation forward?

On the storage side, we might need to make some changes to how we efficiently fetch the set of runs associated with a given sensor, if we are planning to potentially have a bunch of sensors all with the same name in different repositories (@prha you may have some thoughts there)